### ISSUE TITLE
Additional sort options

### ISSUE THREAD

**Issue by Nutzzz** [AlexisL61/JackboxUtility#331]
1. Max Playtime
It's odd that Sort by Player Count exists but not by Sort by Max Playtime, since that data is already filterable.

2. Last Played Date
This data is probably not recorded already (and of course Steam only records this info by pack), but this is my primary suggestion, as I often have a hard time remembering the name of the games.  Since playing Jackbox Games is often situational, it may be quite some time before I open them again, and I imagine I'm not alone in having this problem.

3. Most Played
Once you've decided to start recording this kind of information, then you might also sort by the total number of plays.

4. Total Playtime
This is another possibility, though these games are probably more likely than most to be left sitting idle between play sessions, so I'm not sure about that one.

**Comment by AkiraArtuhaxis** [comment]
I agree with the first three. The last one isn't really necessary if you're already recording the ones that you launch the most. I suppose this varies if you play multiple games in a row with one game, although I doubt there'd be too much change.
Also, the second one should be named "Sort by Last Played", removing the date. The date itself probably wouldn't be in the game info tabs, so this name suits it better.
